src/modm/platform/core/hosted/module.lb

```diff
@@ -47,11 +47,11 @@ def build(env):
         if not env.has_module(":test:architecture"):
             env.template("clock.cpp.in")
 
-    if target.family == "windows":
-        env.log.error("Assertions are not implemented!")
-        if env.has_module(":architecture:delay"):
-            env.log.error("Delay functions are not implemented!")
-    else:
+    # if target.family == "windows":
+    #     env.log.error("Assertions are not implemented!")
+    #     if env.has_module(":architecture:delay"):
+    #         env.log.error("Delay functions are not implemented!")
+    # else:
         env.template("assert.cpp.in")
```

Comment:
This seems odd to me... didn't this change make it so that this code is in a different scope than it used to be?

Reply:
sorry for my lack of python 😅 , looks like it happened to work anyway in this case, I'll fix that